test/simulator/timestamp/src/connection_simulator.cpp

```diff
@@ -211,15 +203,18 @@ connection_simulator::set_timestamp(const std::string &config)
 
     timestamp_manager *ts_manager = &timestamp_manager::get_timestamp_manager();
     std::map<std::string, std::string> config_map;
+    const std::vector<std::string> supported_ops = {
+      "oldest_timestamp", "stable_timestamp", "durable_timestamp", "force"};
```

Comment:
Please A->Z

Reply:
Updated in [6cf2950](https://github.com/wiredtiger/wiredtiger/pull/8372/commits/6cf2950112ff17a0b4f29be84f643d56365d9006)